lib/uksched/include/uk/thread.h

```diff
@@ -54,45 +54,74 @@ typedef void (*uk_thread_fn1_t)(void *) __noreturn;
 typedef void (*uk_thread_fn2_t)(void *, void *) __noreturn;
 
 struct uk_thread {
-	struct ukarch_ctx    ctx;	/**< Architecture context */
-	struct ukarch_ectx *ectx;	/**< Extended context (FPU, VPU, ...) */
-	uintptr_t           tlsp;	/**< Current active TLS pointer */
-	__uptr            uktlsp;	/**< Unikraft TLS pointer */
+	/* Architecture context */
```

Comment:
Give a brief detail of what this context contains. Particularly compared to `ukarch_ectx`.